### DISCUSSION TITLE
Mobile app doesn't stay signed in

### DISCUSSION THREAD

**Discussion by jidanni** [community/community#127625] [Mobile]
Every few days I have to go through the sign in process over again. 

True, I don't have to type in an actual password but still it takes a several button taps. 

The app didn't used to be this way.

**Comment by esahmed21** [comment]
Hi @jidanni 👋 

Could you please share if this happens on Android or iOS and the app version you're using?

Thanks for getting in touch!

**Reply by jidanni** [reply]
I'm using android, app 1.164.

I have set the app to optimize battery though. 

Today was a good day, it didn't get logged out.

**Comment by jidanni** [comment]
Happened again today:

![Screenshot_20240611_143406_GitHub.jpg](https://github.com/community/community/assets/1230959/9d026378-191f-4d1e-a09c-6f2988abcc47)

![Screenshot_20240611_143431_Chrome Beta.jpg](https://github.com/community/community/assets/1230959/d2b8eef7-db28-46e5-80d8-1274393899cb)

**Comment by stevepopovich** [comment]
👋🏼 @jidanni are you still having this issue? Have you tried a fresh reinstall?
